### PR TITLE
[T-6345][15.0][ADD]account_budget_propagate_date: apply the budget dates to budget lines

### PR DESCRIPTION
**Pull request by RogerSans**
This module allows you to apply the budget dates to budget lines.
T-6345

@HaraldPanten
@ValentinVinagre

### CODE REVIEW

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
It's not correct. Check a couple of things:

1- In the Contributors, my email is not correct.

2- The following case cannot be carried out:

- I have a budget in which the start date is 01/01/2025 and the end date is 12/31/2025.
- On the lines, the start and end dates are 01/01/2025 and 12/31/2025 respectively.
- Change the budget date range to 01/01/2026 (start) and 12/31/2026 (end).
- When clicking on applying dates on budget lines, an error appears.

See these screenshots.

Captura de pantalla de 2024-07-13 17-09-41

Captura de pantalla de 2024-07-13 17-09-52

**Comment by RogerSans** [comment]
> It's not correct. Check a couple of things:
> 
> 1- In the Contributors, my email is not correct.
> 
> 2- The following case cannot be carried out:
> 
> * I have a budget in which the start date is 01/01/2025 and the end date is 12/31/2025.
> * On the lines, the start and end dates are 01/01/2025 and 12/31/2025 respectively.
> * Change the budget date range to 01/01/2026 (start) and 12/31/2026 (end).
> * When clicking on applying dates on budget lines, an error appears.
> 
> See these screenshots.
> 
> Captura de pantalla de 2024-07-13 17-09-41
> 
> Captura de pantalla de 2024-07-13 17-09-52

Done.

File: account_budget_propagate_date/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Account Budget Propagate Date
+=============================
+
+This module allows you to change the default font size of the invoice reports.
```
Comment (HaraldPanten) [inline account_budget_propagate_date/README.rst]:
```suggestion
This module allows you to change the default font size of the invoice reports.
```

This is not correct.

**Review by HaraldPanten — CHANGES_REQUESTED** [review]

File: account_budget_propagate_date/README.rst (hunk 1 above)
Comment (RogerSans) [reply account_budget_propagate_date/README.rst]: Solved

**Review by HaraldPanten — APPROVED** [review]
Functional review.

@manuelregidor could you review? THX

File: account_budget_propagate_date/README.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Contributors
+~~~~~~~~~~~~
+
+* Harald Panten <harald.panten@sygel.es>
+* Valentín Vinagre <valentin.vinagre@sygel.es>
+* Roger Sans <roger.sans@sygel.es>
+
+
+Maintainer
```
Comment (manuelregidor) [inline account_budget_propagate_date/README.rst:58]: The sections related to the sygel repository in which this modules is gonna be included are missing.

File: account_budget_propagate_date/models/crossovered_budget.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from odoo import models
```
Comment (manuelregidor) [inline account_budget_propagate_date/models/crossovered_budget.py:4]: Copyright info is missing.

**Review by manuelregidor — CHANGES_REQUESTED** [review]
Please, change the PR title (task code is missing)

File: account_budget_propagate_date/README.rst (hunk 2 above)
Comment (RogerSans) [reply account_budget_propagate_date/README.rst:58]: Added

File: account_budget_propagate_date/models/crossovered_budget.py (hunk 3 above)
Comment (RogerSans) [reply account_budget_propagate_date/models/crossovered_budget.py:4]: Added

**Review by manuelregidor — APPROVED** [review]
Technical review. LGTM

**Comment by manuel-florido** [comment]
@RogerSans 

Addons table not updated, fix it pls

**Pull request merged**